src/commons/sideContent/SideContentContestVotingContainer.tsx

```diff
@@ -19,30 +19,29 @@ type StateProps = {
 /**
  * Container to separate behaviour concerns from rendering concerns
  * Stores component-level voting ranking state
+ * Removed minScore and maxScore due to invalid scores no longer being possible
  */
 const SideContentContestVotingContainer: React.FunctionComponent<
   SideContentContestVotingContainerProps
 > = props => {
   const { canSave, contestEntries, handleSave, handleContestEntryClick } = props;
   const [isValid, setIsValid] = useState<boolean>(true);
   const [votingSubmission, setVotingSubmission] = useState<ContestEntry[]>([]);
-  const minScore = 11 - contestEntries.length;
-  const maxScore = 10;
+  // const minScore;
+  // const maxScore;
 
   useEffect(() => {
     setVotingSubmission(contestEntries);
   }, [contestEntries]);
 
   /**
-   * Validates input value and clamps the value within the min-max range [1, number of entries].
+   * Validates input value.
    * @param votingSubmission voting scores by user for each contest entry.
    * @returns boolean value for whether the scores are within the min-max range.
+   * Removed minScore and maxScore constraints.
```

Comment:
Ditto. This comment is not needed, since all the code related to `minScore` and `maxScore` should be removed, not commented.

Reply:
Resolved in 7a5b1bb5afe0fe7524c3dea5ffc9df35a98c205a, closing this thread.